### PR TITLE
Made changes to tabs and filter UI

### PR DESCRIPTION
**Pull request by Myrl**
Everything should work fine. I changed the tab layout, and moved the filter buttons.

### CODE REVIEW

**Comment by darklordabc** [comment]
I'll test soon.

**Comment by darklordabc** [comment]
Pretty good, just a few things.
1. Rename "Skill" to "Skills" and "Hero" to "Heroes"
2. Rearrange the order of the buttons: Should be "Preset Builds", "Skills" then "Heroes"
3. Make the buttons bigger so its more like the original mock-up.
4. The "Filter by name:" and its associated search box, moves slightly when you switch between Heroes and Skills, its a tiny bit lower on Heroes. It should be in a identical position for both of those tabs.

**Comment by darklordabc** [comment]
Also, the text for the buttons, should be in the localization file.

**Comment by TideSofDarK** [comment]
Working on fixed right now.

**Comment by Myrl** [comment]
@TideSofDarK I already finished it.

**Comment by Myrl** [comment]
Wait, the localization seems to be broken.

**Comment by TideSofDarK** [comment]
Reworked tab structure so its more clean currently.
Needs testing.

File: src/content/panorama/layout/custom_game/game_setup.xml [hunk 1]
```diff
@@ -247,15 +262,14 @@
                                 </Panel>
                             </Panel>
                         </Panel>
+                        
                         <Panel id="pickingPhaseSkillTab" class="pickingPhaseTab">
-                            <Label text="Back" link="pickingPhaseMainTab" class="heroBuildMenuButton addMarginBelow" />
-
-                            <Label text="Filter by name:" class="buildingHelperInfoText" />
+                            <Label text="Filter by name:" class="buildingHelperInfoText addMarginTop" />
```
Comment (darklordabc) [inline src/content/panorama/layout/custom_game/game_setup.xml]: Shouldn't this be in localization.

**Comment by darklordabc** [comment]
Looks pretty good to me. LGTM

**Comment by TideSofDarK** [comment]
LGTM

**Comment by Myrl** [comment]
@TideSofDarK Thanks for redoing the tab structure. I was thinking that it was probably too sensitive to make any changes to that. Guess I was wrong.

**Comment by Yahnich** [comment]
LGTM

**Pull request merged**